### PR TITLE
docs: add vite-plugin-csv

### PR DESCRIPTION
**Pull request by SirwanAfifi**
> Please make sure all the requirements are satisfied, otherwise the PR could be closed without further notice.

## Checklist

- [x] Title as described.
- [x] Make sure you put things in the right category.
- [x] The description of your item should be a sentence with less than 24 words.
- [x] Omit unnecessary words already provided in the context (e.g. `for Vite`, `a Vite plugin`).
- [x] Always add your items to the end of a list.

### Plugins/Tools

- [x] The plugin/tool is working with **Vite 2.0 and onward**.
- [x] The project is Open Source.
- [x] The project follows the [Vite Plugins Conventions](https://vitejs.dev/guide/api-plugin.html#conventions).
- [x] The repo should be at least 30 days old.
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product.

### Starter Templates

- [ ] The starter template is working with **Vite 2.0 and onward**.
- [ ] The documentation is in English.
- [ ] The starter template most provide enough instructions / documentation about how to start up and what's is included.
- [ ] A screenshot or a live demo should be included.
- [ ] The repo should be at least 30 days old.
- [ ] Should be differentiable from the existing starter templates. 

### Apps/Websites

- [ ] The website is available without errors and load in a reasonable amount of time.
- [ ] The website must be Open Source and showing it's using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +237,7 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 - ![v23] [vite-plugin-icons](https://github.com/antfu/vite-plugin-icons) - Access thousands of icons as Vue components.
 - ![v3] [vite-plugin-vuedoc](https://github.com/JasKang/vite-plugin-vuedoc) - Markdown Code block as Vue Preview components.
 - ![v3] [vite-svg-loader](https://github.com/jpkleemans/vite-svg-loader) - Load SVG files as Vue components.
+- [vite-plugin-csv](https://github.com/SirwanAfifi/vite-plugin-csv) Import CSV files in your vite projects
```
Comment (Scrum) [inline README.md:240]: what kind of support vue? 2/3 or only 3?

Comment (SirwanAfifi) [reply README.md]: @Scrum Opps, I should have moved it to `Framework-agnostic Plugins` section, I have just fixed and pushed.

**Review by Scrum — APPROVED** [review]

**Comment by Scrum** [comment]
@SirwanAfifi https://github.com/vitejs/awesome-vite/pull/246/checks?check_run_id=3350157645

**Comment by SirwanAfifi** [comment]
> @SirwanAfifi https://github.com/vitejs/awesome-vite/pull/246/checks?check_run_id=3350157645

It should be fixed now.

**Comment by Scrum** [comment]
@SirwanAfifi https://github.com/vitejs/awesome-vite/pull/246/checks?check_run_id=3350819252

**Comment by patak-js** [comment]
There is already an official rollup plugin that is compatible with for csv https://github.com/rollup/plugins/tree/master/packages/dsv. Is there something that this plugin will offer?
@Scrum @antfu should there be a check box for this in the PR template?

**Comment by antfu** [comment]
Yes, I agree with @patak-js. Giving the fact that this plugin does not offer vite specific features, I think we should advocate the existing `@rollup-plugin/dsv` instead.

**Pull request closed without merging**